### PR TITLE
Fix PAT creation

### LINKED ISSUE #1639: Authorized Applications scopes are hard to parse
Looking at the applications I've granted access to my account, it's hard to see what they can and can't do at a glance.  The strikethrough is classy, but maybe make the struck-through text a lighter grey or bold the other text or add some color.  This was alarming to see at first:

screen shot 2017-04-25 at 2 43 09 pm

### LINKED ISSUE #1874: Token access levels not styled to display access
screen shot 2017-05-24 at 5 17 19 pm

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #1639, closes #1874

screen shot 2017-05-30 at 2 40 30 pm

### CODE REVIEW

**Comment by eatonphil** [comment]
@mparke I am not sure why it's needed. Take a look if you'd like.

**Comment by mparke** [comment]
pull against this branch @eatonphil https://github.com/eatonphil/manager/pull/13

**Comment by eatonphil** [comment]
thanks!
